<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWidget</class>
 <widget class="QWidget" name="ChartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wykres pomiarow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wykres danych pomiarowych</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="parameterLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Parametr: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="timeRangeLabel">
       <property name="text">
        <string>Zakres czasu:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="timeRangeComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Data poczatkowa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="endDateLabel">
       <property name="text">
        <string>Data koncowa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="endDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="chartLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="analyzeButton">
       <property name="text">
        <string>Analizuj dane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveChartButton">
       <property name="text">
        <string>Zapisz wykres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportDataButton">
       <property name="text">
        <string>Eksportuj dane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Powrot do listy czujnikow</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>